website/cue/reference/components/sinks/base/clickhouse.cue

```diff
@@ -298,6 +303,41 @@ base: components: sinks: clickhouse: configuration: {
 		required:    true
 		type: string: examples: ["mytable"]
 	}
+	sql_table_col_def: {
+		description: """
+			The clickhouse table column definition.
```

Comment:
Fixed it across all documentation. Now all ClickHouse naming should be valid. The only question is about naming the variables in the source code. Do we need to update their naming to `ClickHouse` variant too? IMO, at least not in this PR - these changes will be too huge and without a visible profit for our users since these changes will be "invisible".